domains/deepak.is-cool.dev.json

```diff
@@ -10,8 +10,7 @@
     },
 
     "record": {
-        "MX": ["mx1.forwardemail.net", "mx2.forwardemail.net"],
-        "TXT": ["forward-email=hello:gohildeepak2003@gmail.com", "v=spf1 a mx include:spf.forwardemail.net include:_spf.google.com -all"]
+        "NS": ["ns1.hostry.com", "ns2.hostry.com", "ns3.hostry.com", "ns4.hostry.com"]
```

Comment:
Please provide a detailed and reasonable explanation for the use of NS records, as per outlined in the readme.md file.

Reply:
I kindly request your assistance in adding an NS (Name Server) record for the subdomain. This change will allow me to manage multiple records independently, eliminating the need for frequent pull requests. I assure you that I will strictly follow the repository guidelines. Your support in this matter would greatly enhance efficiency and contribute to a smoother workflow. Thank you for your consideration.